Userland/Libraries/LibCore/File.cpp

```diff
@@ -85,8 +93,10 @@ int File::open_mode_to_options(OpenMode mode)
         flags |= O_EXCL;
     if (!has_flag(mode, OpenMode::KeepOnExec))
         flags |= O_CLOEXEC;
+#if !defined(AK_OS_WINDOWS)
     if (has_flag(mode, OpenMode::Nonblocking))
         flags |= O_NONBLOCK;
+#endif
```

Comment:
dropping this flag on the floor seems problematic. We should at least warnln or dbgln (or crash?) to let developers know that they aren't going to be getting non-blocking behavior on that file.

If we actually need non-blocking behavior on non-sockets (I believe the way request server works right now needs this?) we'll need to come up with some non-blocking I/O alternatives. Would be easier to track those down with a log message   or crash.

Reply:
Added dbgln. Currently nobody uses this flag.